### PR TITLE
[Keyboard] AkiS 

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Add AkiS keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add AkiS keyboard

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xelus/akis/rules.mk [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/xelus/akis/rules.mk:11]:
```suggestion
```

File: keyboards/xelus/akis/rules.mk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (drashna) [inline keyboards/xelus/akis/rules.mk:29]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xelus/akis/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+AKI-S
+===
```
Comment (drashna) [inline keyboards/xelus/akis/readme.md:2]:
```suggestion
# AKI-S
```

File: keyboards/xelus/akis/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+A 65% Keyboard by Thic Thock.
+
+Keyboard Maintainer: [Xelus22](https://github.com/Xelus22)  
+Hardware Supported: AkiS 
+Hardware Availability: Custom keyboard group buys
```
Comment (drashna) [inline keyboards/xelus/akis/readme.md:8]:
```suggestion
* Keyboard Maintainer: [Xelus22](https://github.com/Xelus22)  
* Hardware Supported: AkiS 
* Hardware Availability: Custom keyboard group buys
```

File: keyboards/xelus/akis/readme.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/xelus/akis/readme.md:10]: And could you add instructions on how to jump to the bootloader?

File: keyboards/xelus/akis/rules.mk [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
```
Comment (drashna) [inline keyboards/xelus/akis/rules.mk:13]: May want to disable this, to reduce compiled size.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xelus/akis/rules.mk (hunk 6 above)
Comment (Xelus22) [reply keyboards/xelus/akis/rules.mk:13]: I have LTO enabled I think that should be enough?

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**